CHANGES.md

```diff
@@ -4,7 +4,7 @@ Release Notes.
 
 8.12.0
 ------------------
-
+* Fix `Shenyu plugin` IgnoredTracerContext traceId NPE(Use with the apm-trace-ignore-plugin plugin will trigger)
```

Comment:
```suggestion
* Fix `Shenyu plugin`'s NPE in reading read trace ID when IgnoredTracerContext is used in the context.
```

Reply:
IgnoredTracerContext could be activated in many cases, such as ignore-plugin, sampling policy activated, and agent disconnected with the OAP server.

Reply:
> IgnoredTracerContext could be activated in many cases, such as ignore-plugin, sampling policy activated, and agent disconnected with the OAP server.

Understood, not enough consideration at the time